CodePush.js

```diff
@@ -294,6 +294,7 @@ function sync(options = {}, syncStatusChangeCallback, downloadProgressCallback)
       })
       .catch((error) => {
         syncStatusChangeCallback(CodePush.SyncStatus.UNKNOWN_ERROR);
+        log(error.message);
```

Comment:
nit: don't know if it matters, but may be log error before calling syncStatusChangeCallback, if for some reason callback itself crashes we won't have this logged?